src/components/pages/UserProfilesPage.tsx

```diff
@@ -133,30 +132,16 @@ const UserProfilesPage: FunctionComponent = () => {
     return (
         <div ref={element}>
             <div className='content-primary'>
-                <div className='verticalSection verticalSection-extrabottompadding'>
```

Comment:
It looks like we are losing the extra padding here. Does that matter?

Reply:
No need.